crux-mir/lib/libstd/sys_common/poison.rs

```diff
@@ -20,7 +20,11 @@ pub struct Flag {
 
 impl Flag {
     pub const fn new() -> Flag {
-        Flag { failed: AtomicBool::new(false) }
+        #[cfg(feature = "opaque-poison-atomics")]
+        { Flag { failed: AtomicBool::new_unmodeled(false) } }
+
+        #[cfg(not(feature = "opaque-poison-atomics"))]
+        { Flag { failed: AtomicBool::new_unmodeled(false) } }
```

Comment:
This has the same code in both `cfg` cases - is that correct?

Reply:
No, good catch. I caught another location in liballoc/sync.rs as well.